plugins/spu2-x/src/Windows/Config.cpp

```diff
@@ -294,6 +305,19 @@ BOOL CALLBACK ConfigProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
                     EndDialog(hWnd, 0);
                     break;
 
+                case IDC_OUTPUT:
+                    if (wmEvent == CBN_SELCHANGE) {
+                    const int module = (int)SendMessage(GetDlgItem(hWnd, IDC_OUTPUT), CB_GETCURSEL, 0, 0);
```

Comment:
if and it's contents have the same indentation level:
```cpp
                    if (wmEvent == CBN_SELCHANGE) {
                    const int module = (int)SendMessage(GetDlgItem(hWnd, IDC_OUTPUT), CB_GETCURSEL, 0, 0);
```
And `if` brace ends at `} break;`, while looking at the next case it seems like the entire case body (including `if`) was supposed to be inside a case block.